src/main/java/flaxbeard/steamcraft/block/BlockBoiler.java

```diff
@@ -178,15 +179,15 @@ public IIcon getIcon(IBlockAccess block, int x, int y, int z, int side) {
             return side == 1 ? this.field_149935_N : this.blockIcon;
         }
 
-
     }
 
     @Override
     public IIcon getIcon(int side, int meta) {
         if (meta == 0) {
             meta = 3;
         }
-        return side == 1 ? this.field_149935_N : (side == 0 ? this.field_149935_N : (side != meta ? this.blockIcon : this.boilerOffIcon));
+        return side == 1 ? this.field_149935_N
+                : (side == 0 ? this.field_149935_N : (side != meta ? this.blockIcon : this.boilerOffIcon));
```

Comment:
This is just kind of ugly. A better thing would be to store the IIcon rather than nesting ternaries
```java
IIcon icon = side == meta ? boilerOffIcon : blockIcon;
IIcon blahblahblah = side == 0 ? field_149935_N : icon;
return side == 1 ? field_149935_n : blahblahblah;
```
```